src/Microsoft.DotNet.Build.Tasks.net45/Microsoft.DotNet.Build.Tasks.net45.csproj

```diff
@@ -38,6 +39,7 @@
     <TargetingPackReference Include="System.Threading.Tasks" />
     <TargetingPackReference Include="System.Xml" />
     <TargetingPackReference Include="System.Xml.Linq" />
+    <TargetingPackReference Include="System.Windows.Forms" />
```

Comment:
I think Wes's point is using System.Windows.Forms would not exist if msbuild will use net core and not the full framework. This means you just want to avoid using ResXResourceWriter and you may write the generated resw manually using Xml writer.